vita3k/gui/src/archive_install_dialog.cpp

```diff
@@ -33,7 +32,7 @@ static bool delete_archive_file;
 static std::string state, type, title;
 static std::map<fs::path, std::vector<ContentInfo>> contents_archives;
 static std::vector<fs::path> invalid_archives;
-static nfdchar_t *archive_path;
+static std::filesystem::path archive_path = "";
```

Comment:
Can you use fs::filesystem::... instead for all the fs filesystem objects?

Reply:
Why? `fs::filesystem` is actually a nickname for `boost::filesystem` which is at the same time a C++ filesystem implementation previous to the creation of the official C++17 `std::filesystem`? Also `std::filesystem` plays way better with C++ and C code due to using all C++ types instead of custom Boost types.

Reply:
As you want, but everything else uses fs::filesystem so it was more for consistency.

Reply:
Is there any particular reason why the project still uses `boost::filesystem` instead of `std::filesystem`?

Reply:
#1912